catalog-info.yaml

```diff
@@ -0,0 +1,17 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "frontend-app-gradebook"
+  description: "The frontend (MFE) for Open edX Gradebook"
+  links:
+    - url: "https://github.com/openedx/frontend-app-gradebook"
+      title: "Frontend App Gradebook"
+      icon: "Web"
```

Comment:
You don't need to provide a link to the code, that is done automatically in backstage.

Reply:
thanks